waveshare2in13v2/waveshare213v2.go

```diff
@@ -21,6 +20,32 @@ import (
 	"periph.io/x/host/v3/rpi"
 )
 
+// Commands
+const (
+	driverOutputControl            byte = 0x01
+	gateDrivingVoltageControl      byte = 0x03
+	sourceDrivingVoltageControl    byte = 0x04
+	dataEntryModeSetting           byte = 0x11
+	swReset                        byte = 0x12
+	masterActivation               byte = 0x20
+	displayUpdateControl1          byte = 0x21
+	displayUpdateControl2          byte = 0x22
+	writeRAMBW                     byte = 0x24
+	writeRAMRed                    byte = 0x26
+	writeVcomRegister              byte = 0x2C
+	writeLutRegister               byte = 0x32
+	setDummyLinePeriod             byte = 0x3A
+	setGateTime                    byte = 0x3B
+	borderWaveformControl          byte = 0x3C
+	unknownCommand37               byte = 0x37
```

Comment:
Was it listed in the datasheet or not?

Reply:
Ah, this was a leftover from an earlier revision. No, this particular comment isn't documented anywhree. I removed the constant again.